zen/core/context.py

```
@@ -170,122 +162,121 @@ def _load_genesis_docs(self):
+                self.project_context["zenos_version"] = self._get_zenos_version()
         except Exception as e:
             console.print(f"[yellow]Warning: Could not load project context: {e}[/yellow]")
-    
+
     def _load_git_context(self):
         """Load git information."""
         try:
             # Check if we're in a git repo
             result = subprocess.run(
-                ['git', 'rev-parse', '--git-dir'],
+                ["git", "rev-parse", "--git-dir"],
```

Ruff (0.15.6)

[error] Starting a process with a partial executable path

(S607)